.github/workflows/build-container.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-          test: ["cli", "state", "multi_gateway"]
+          test: ["cli", "state", "multi_gateway", "spdk_exception"]
```

Comment:
Nit: rather than after the bug name, I would name the test suite after:
* The python module (in this case `server`),
* The stage at which this happens (`start_up`)

That way you can add further related tests to that suite without having to rename it.